packages/xo-web/src/xo-app/logs/backup-ng/log-alert-header.js

```diff
@@ -66,6 +67,31 @@ export default decorate([
       formattedLog: (_, { log }) => JSON.stringify(log, null, 2),
       jobFailed: (_, { log = {} }) =>
         log.status !== 'success' && log.status !== 'pending',
+      reportBugProps: ({ formattedLog }, { log = {}, jobs }) => {
+        const props = {
+          size: 'small',
+          title: 'Backup job failed',
+        }
+        if (currentPlan === XOA_PLAN_SOURCES) {
+          props.message = `\`\`\`json\n${formattedLog}\n\`\`\``
+        } else {
+          const formattedDate = ifDef(log.start, formatDate)
+          props.files = [
+            {
+              content: createBinaryFile(formattedLog),
+              name: `${formattedDate} - backup.log`,
```

Comment:
What is the format of `formattedLog`?

If JSON, it would be better to add the correct extension to have the proper syntax highlighting when opened.